src/modules/Dropdown/Dropdown.js

```diff
@@ -134,6 +134,12 @@ export default class Dropdown extends Component {
     /** Define whether the highlighted item should be selected on blur */
     selectOnBlur: PropTypes.bool,
 
+    /** Make the dropdown options searchable (`boolean` or `function(options, searchQuery) => filteredOptions`) */
```

Comment:
I see. The function signature could be handy though (as long as there is no other way to see this). Do I add that temporarily to the comment, or totally not?

Something like: ```Make the dropdown options searchable by substring matching (default) or with a custom search function `(options, searchQuery) => filteredOptions` ```, or do I leave off the function signature?